spec/support/tasks.cr

```diff
@@ -10,6 +10,10 @@ class Some::Other::Task < LuckyCli::Task
   summary "bar"
   name "my.custom_name"
 
+  def help_message
+    "Run with 'lucky my.custom_name'"
```

Comment:
Maybe "Custom help message" just so it is extra clear the is not the default one?